.flake8

```diff
@@ -1,5 +1,4 @@
 [flake8]
 max-line-length = 120
-select=E901,E999,F821,F822,F823,I100,I101,I201,I202,T001,T002,T003,T004
-ignore=E203,W503,C0330
-application-import-names=tribler,run_tribler
+ignore=E127, E128, E131, E731, E722, E501, F401, F841,  F722 , I202
```

Comment:
It seems it is not correct to specify both `select` and ìgnore`. In stat case, the `ignore` section just got ignored.
Also, the previous `select` list of enabled warnings did not include some useful warnings, for example:

```
./src\tribler\core\components\libtorrent\download_manager\download.py:714:5:
F811 redefinition of unused 'get_upload_mode' from line 705

./src\tribler\core\components\libtorrent\download_manager\download.py:718:5:
F811 redefinition of unused 'set_upload_mode' from line 709

./src\tribler\gui\tests\test_error_handler.py:68:1:
F811 redefinition of unused 'test_gui_core_connect_timeout_error' from line 59
```
I think the best way is to define extend-ignore list with errors we want to ignore:
```suggestion
extend-ignore =
  I202, # Additional newline in a group of imports
  E127, # continuation line over-indented for visual indent
  E128, # continuation line under-indented for visual indent
  E131, # continuation line unaligned for hanging indent
  F722, # syntax error in forward annotation '20s'
  E731, # do not assign a lambda expression, use a def
  F401, # Pylint-checked, module imported but unused
  F841, # Pylint-checked, local variable is assigned to but never used
  E722, # Pylint-checked, do not use bare 'except'
  E501 # Pylint-checked, line too long
```

I also added comments to error codes